sig-cluster-lifecycle/postmortems/kubeadm-1.6.md

```diff
@@ -70,7 +70,7 @@ initialize a new cluster with the 1.6.0 release
 
 - There was a lot of rushing to get the release ready before KubeCon EU, causing
   a shortened timeframe for RC and release, lowered bandwidth for SIG members,
-  and the last SIG meeting before the release to be cancelled, which decreased
+  and the last SIG meeting before the release to be canceled, which decreased
```

Comment:
This is the British spelling, and can be removed from this PR.

Reply:
👌， thanks for your tips